docs/iris/src/whatsnew/contributions_1.9/newfeature_2015-10-19_newaxis_cube_indexing.txt

```diff
@@ -0,0 +1 @@
+Cube indexing now support newaxis.
```

Comment:
Thanks @rhattersley. Agree with all of the above. I didn't spend a huge amount of time simply because this is edited by hand at release anyway - I was working on ensuring the message was clear, even if the grammar wasn't :wink: 

I'll get that patched up today.